generator/src/main/java/com/linkedin/pegasus/generator/FileFormatDataSchemaParser.java

```diff
@@ -92,8 +92,10 @@ public FileFormatDataSchemaParser(String resolverPath, DataSchemaResolver schema
               while (entries.hasMoreElements())
               {
                 final JarEntry entry = entries.nextElement();
-                if (!entry.isDirectory() && entry.getName().endsWith(_schemaParserFactory.getLanguageExtension()) &&
-                    (entry.getName().startsWith(_schemaResolver.getSchemasDirectoryName().getName()) || isExtensionEntry(entry)))
+                if (!entry.isDirectory() &&
+                    entry.getName().endsWith(_schemaParserFactory.getLanguageExtension()) &&
+                    (entry.getName().startsWith(SchemaDirectoryName.PEGASUS.getName()) ||
```

Comment:
I don't see why "parseSources" should load from both locations.
Based on my understanding: When SchemaDirectoryName == EXTENSION, the ExtensionSchemaResolver will be configured to resolve from both locations.

And parseSource should only parse the extension schemas, right?

Reply:
The reason for doing this change is for users who don't want to create two parsers to call parse sources twice.
The other reason is that when I add the condition in this FileFormatDataSchemaParser, I had the pegasus base entry in mind.
I also feel that extension schema only also make sense.

For the other question that resolve from both location, it is true, and it is in my existing unit test cases.

Reply:
After offline discussion, I removed the support of parsing pegasus PDL by ExtensionSchemaResolver